cmd/res/configuration.toml

```diff
@@ -48,9 +57,17 @@ PublishTopicPrefix = 'edgex/events' # /<device-profile-name>/<device-name>/<sour
   Retained = "false"
   AutoReconnect = "true"
   ConnectTimeout = "5" # Seconds
+  SkipCertVerify = "false" # Only used if Cert/Key file or Cert/Key PEMblock are specified  [MessageQueue.Optional]
+  # Default MQTT Specific options that need to be here to enable environment variable overrides of them
+  # Client Identifiers
+  ClientId = "device-random"
```

Comment:
`device-modbus`

Reply:
Humm, good catch!

Reply:
ahh, it was duplicated. Removed that extra setting.